### ISSUE TITLE
COMMAND[....] does not longer replace &player and other variables

### ISSUE THREAD

**Issue by timbru31** [gravitylow/AntiCheat#156]
When a command is executed (e.g. COMMAND[warn &player] ) the &player variable is now ignored.
This is due the change that multiple commands can be executed now.

Caused by this commit:
https://github.com/h31ix/AntiCheat/commit/a89fc9786312d6cfce141911cbc2431f9c86ca33

The username (&player) is not longer replaced as before:
https://github.com/h31ix/AntiCheat/blob/master/src/main/java/net/h31ix/anticheat/manage/UserManager.java#L134

Hope you can fix this fast :)

**Comment by gravitylow** [comment]
In that commit, this line still replaces those variables: https://github.com/h31ix/AntiCheat/commit/a89fc9786312d6cfce141911cbc2431f9c86ca33#L0R1055 

In the string 'event', '&player' and '&world' are still parsed before the loop through the commands even occurs.

**Comment by timbru31** [comment]
You are right, but this check is for chat spamming; I don't know your source code that good, but the execute() code is called onHigh and onMed alerts, if I looked right at the source.
And there is no replacement of the variables on the execute check (anymore).
Test it yourself, latest build (#408) and use a command action on high level, won't replace it.

Cheers

**Sir edit:**
https://github.com/h31ix/AntiCheat/commit/a89fc9786312d6cfce141911cbc2431f9c86ca33#L1L135
This line was removed, containing ".replaceAll("&player", name).replaceAll("&world", user.getPlayer().getWorld().getName());"
But the Utilities.getCommands() does not replace these variables again
